src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -210,56 +145,57 @@ private string ClearHexString(string hexString)
             }
         }
 
+        private bool IsHex(string str) => str.Length % 2 == 0 && str.All(c => (c >= '0' && c <= '9') || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F'));
+
```

Comment:
This will always be `false` because of the `prefix` `0x` having the `x` in it.

Reply:
It is by design, only for hexstring without '0x'

Reply:
dotnet format passed with no changes